chain/jsonrpc/src/lib.rs

```diff
@@ -1048,6 +1051,26 @@ impl JsonRpcHandler {
         }
         None
     }
+
+    /// Returns the best window for maintenance in current epoch for the specify account
```

Comment:
s/specify/specified.

let's expand this comment a little: mention what maintenance window is (I guess the time when the given account doesn't produce any blocks or chunks, right?)

Reply:
Yes, exactly.  Thanks for the review